src/common/syncjournaldb.cpp

```diff
@@ -1376,6 +1378,13 @@ bool SyncJournalDb::updateLocalMetadata(const QString &filename,
     query->bindValue(2, inode);
     query->bindValue(3, modtime);
     query->bindValue(4, size);
+    query->bindValue(5, lockInfo._locked ? 1 : 0);
+    query->bindValue(6, lockInfo._lockOwnerDisplayName);
+    query->bindValue(7, lockInfo._lockOwnerId);
```

Comment:
@claucambra You have `query->bindValue(7, lockInfo._lockOwnerId);` here, while you also have `lockOwnerId=?8` in the query. Shouldn't this be  `query->bindValue(8, lockInfo._lockOwnerId);`?

Reply:
Yeah, apologies -- fixed the order